### DISCUSSION TITLE
Instructions to create own LXC scripts

### DISCUSSION THREAD

**Discussion by Trblz42** [tteck/Proxmox#2296] [General]
I've been using some of the scripts and eager to create my own / add to the repo.

I could not found complete instructions on how to reproduce similar scripts, so here are my notes for feedback: 

Assuming LXC setup at minimum two scripts need to be created

In Proxmox / CT / <script name>.sh  which is used to start the 

- install process, 
- define LXC parameters

In Proxmox / install / <script name>_install.sh to list the steps within the container to 

- install dependencies (apt get), 
- install services,  
- create default configuration files within container 
- as well as root password access (motd_ssh & customize)

what am i missing?

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/blob/main/CODE-AUDIT.md

**Reply by Trblz42** [reply]
i missed this document somehow... oops.

**Reply by tteck** [reply]
I made it a little easier to read.

**Reply by tteck** [reply]
However, you are correct that when contributing new applications to this repository, the `*.sh` and `*-install.sh` scripts are the only components that need to be constructed.
